packages/devtools_app/lib/src/theme.dart

```diff
@@ -25,45 +25,62 @@ ThemeData themeFor({
 }
 
 ThemeData _darkTheme(IdeTheme ideTheme) {
-  final theme = ThemeData.dark();
-  final background = isValidDarkColor(ideTheme?.backgroundColor)
-      ? ideTheme?.backgroundColor
-      : theme.canvasColor;
-
-  return theme.copyWith(
+  return _baseTheme(
+    theme: ThemeData.dark(),
+    ideTheme: ideTheme,
     primaryColor: devtoolsGrey[900],
-    primaryColorDark: devtoolsBlue[700],
-    primaryColorLight: devtoolsBlue[400],
     indicatorColor: devtoolsBlue[400],
-    accentColor: devtoolsBlue[400],
-    backgroundColor: devtoolsGrey[600],
-    canvasColor: background,
-    toggleableActiveColor: devtoolsBlue[400],
     selectedRowColor: devtoolsGrey[600],
-    buttonTheme: theme.buttonTheme.copyWith(minWidth: buttonMinWidth),
-    scaffoldBackgroundColor: background,
-    colorScheme: theme.colorScheme.copyWith(background: background),
   );
 }
 
 ThemeData _lightTheme(IdeTheme ideTheme) {
-  final theme = ThemeData.light();
+  return _baseTheme(
+    theme: ThemeData.light(),
+    ideTheme: ideTheme,
+    primaryColor: devtoolsBlue[600],
+    indicatorColor: Colors.yellowAccent[400],
+    selectedRowColor: devtoolsBlue[600],
+  );
+}
+
+ThemeData _baseTheme({
+  @required ThemeData theme,
+  @required IdeTheme ideTheme,
+  @required Color primaryColor,
+  @required Color indicatorColor,
+  @required Color selectedRowColor,
+}) {
   final background = isValidLightColor(ideTheme?.backgroundColor)
       ? ideTheme?.backgroundColor
       : theme.canvasColor;
   return theme.copyWith(
-    primaryColor: devtoolsBlue[600],
+    primaryColor: primaryColor,
+    indicatorColor: indicatorColor,
+    selectedRowColor: selectedRowColor,
+    // Same values for both light and dark themes.
     primaryColorDark: devtoolsBlue[700],
     primaryColorLight: devtoolsBlue[400],
-    indicatorColor: Colors.yellowAccent[400],
     accentColor: devtoolsBlue[400],
     backgroundColor: devtoolsGrey[600],
-    canvasColor: background,
     toggleableActiveColor: devtoolsBlue[400],
-    selectedRowColor: devtoolsBlue[600],
-    buttonTheme: theme.buttonTheme.copyWith(minWidth: buttonMinWidth),
+    canvasColor: background,
     scaffoldBackgroundColor: background,
     colorScheme: theme.colorScheme.copyWith(background: background),
+    buttonTheme: theme.buttonTheme.copyWith(
```

Comment:
Specifying the buttonTheme shouldn't be needed. To change the minimum size of TextButtons, specify minimumSize for TextButtonTheme below.

Reply:
done. Is there a way to set the default button height? I only saw a way to set the min, so had to enforce button height by wrapping certain buttons in SizedBoxes throughout devtools

Reply:
You're right, currently there's only support for specifying a minimumSize. Using SizedBox is OK for forcing a particular height.  If you think we should add a fixedSize ButtonStyle parameter to ButtonStyle and the button themes, please file an issue.  It seems reasonable to support that, and it's true that we used to.

Reply:
filed https://github.com/flutter/flutter/issues/73741